Gemfile

```diff
@@ -5,10 +5,17 @@ gem 'rake'
 gem 'sinatra'
 gem 'sinatra-activerecord'
 gem 'sinatra-contrib'
+gem 'sinatra-flash'
 
 group :test, :development do
   gem 'database_cleaner'
   gem 'capybara'
   gem 'pry'
   gem 'rspec'
 end
+
+group :test do
+  gem "shoulda"
+  gem "shoulda-matchers", "~> 2.4.0", require: false
+  gem "valid_attribute"
+end
```

Comment:
Now that I'm thinking about it, `pry` is probably the only requirement that needs to be in the `:test, :development` block. Could you move database_cleaner, capy, and rspec to the `:test` block, please?